packages/next-app/src/contexts/StableSwapContext.tsx

```diff
@@ -78,16 +102,69 @@ export function StableSwapProvider({ children }: PropsWithChildren) {
     setVariantData(convertGqlToAnalysisData(poolData));
   }
 
+  function numberToBigNumber(number: number, decimals = 18) {
+    return parseFixed(number.toString(), decimals);
+  }
+
+  function numberToOldBigNumber(number: number, decimals = 18) {
+    return bnum(number.toFixed(decimals));
+  }
+
+  function preparePoolPairData({
+    indexIn,
+    indexOut,
+    swapFee,
+    allBalances,
+    amp,
+  }: {
+    indexIn: number;
+    indexOut: number;
+    swapFee: number;
+    allBalances: number[];
+    amp: number;
+  }) {
+    const allBalancesOldBn = allBalances.map((balance) =>
+      numberToOldBigNumber(balance)
+    );
+    const allBalancesBn = allBalances.map((balance) =>
+      numberToBigNumber(balance)
+    );
+    return {
+      id: "0x",
+      address: "0x",
+      poolType: 1,
+      tokenIn: "0x",
+      tokenOut: "0x",
+      balanceIn: numberToBigNumber(allBalances[indexIn]),
+      balanceOut: numberToBigNumber(allBalances[indexOut]),
+      swapFee: numberToBigNumber(swapFee, 18),
+      allBalances: allBalancesOldBn,
+      allBalancesScaled: allBalancesBn,
+      amp: numberToBigNumber(amp, 3),
+      tokenIndexIn: indexIn,
+      tokenIndexOut: indexOut,
+      decimalsIn: 6,
+      decimalsOut: 18,
+    } as StablePoolPairData;
+  }
+
```

Comment:
We create this because the pools can actually exist if we import, but they can be made up and the StableMath expects data from a real pool